pkg/pillar/Dockerfile.in

```diff
@@ -35,6 +35,7 @@ ADD ./  /pillar/
 
 # go vet/format and go install
 WORKDIR /pillar
+RUN patch -p1 < patch-arm-poll.diff
 RUN [ -z "$GOARCH" ] || export CC=$(echo /*-cross/bin/*-gcc) ;\
```

Comment:
Note that yetus complains about the "=" on this line.

Reply:
yeah, well -- that's an old line -- ignoring